pkg/cgroup/cgroup_stats.go

```diff
@@ -17,84 +17,110 @@ limitations under the License.
 package cgroup
```

Comment:
we'd better have one ore more interfaces for this package and follow https://github.com/sustainable-computing-io/kepler/blob/main/CONTRIBUTING.md#macos to make mac unit test passed.

Reply:
Fixed that. Thanks.